.github/workflows/pregen_all.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python3
 
 import argparse
+from pathlib import Path
 import subprocess
 import sys
-from pathlib import Path
```

Comment:
Doesn't isort default to sorting `from` imports after module imports?

Reply:
Idk. I didn't use isort for this. Again, someone has to add it to wpiformat, cuz I don't want to run it manually every time.